auth-web/src/store/modules/org.ts

```diff
@@ -242,4 +245,10 @@ export default class OrgModule extends VuexModule {
     const response = await OrgService.getOrgInvitations(this.context.state['currentOrganization'].id, 'PENDING')
     return response.data && response.data.invitations ? response.data.invitations : []
   }
+
+  @Action({ rawError: true })
+  public async addOrgProducts (productsRequestBody: ProductsRequestBody): Promise<Products> {
```

Comment:
Could we rename this to be `addProductsToOrg`?

Reply:
Will do